src/main/angular/app/graph/graph.component.ts

```diff
@@ -43,9 +44,11 @@ export class GraphComponent implements OnInit {
 
   constructor(
     private dataService: DataService,
-    private graphProcessor: GraphProcessorService
+    private graphProcessor: GraphProcessorService,
+    private errorMessageService: ErrorMessageService
   ) {
     this.shouldShowChart = false;
+    1;
```

Comment:
is this required?

Reply:
Not sure how or why this is here :)

I'm surprised tslint didn't pick it up.